config.yaml

```
@@ -50,7 +52,7 @@ multiqc_report: True
 fastp:
     extra: ""
 remove_host:
-    db_path: ""              # [Required] Path to folder containing a Kraken2 database with host sequences (taxo.k2d, etc.)
+    db_path: "/db/kraken2/kraken2_GRCh38"              # [Required] Path to folder containing a Kraken2 database with host sequences (taxo.k2d, etc.)
```

```suggestion
    db_path: ""              # [Required] Path to folder containing a Kraken2 database with host sequences (taxo.k2d, etc.)
```
The default path should preferably be empty when this reaches the default branch